docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -26,18 +26,18 @@ The following table lists the available GX Cloud Expectations.
 | Distribution       | `expect_column_z_scores_to_be_less_than`                  | Expect the Z-scores of a column's values to be less than a given threshold.                                                            |
 | Distribution       | `expect_column_stdev_to_be_between`                       | Expect the column standard deviation to be between a minimum value and a maximum value.                                                |
 | Distribution       | `expect_column_sum_to_be_between`                         | Expect the column sum to be between a minimum value and a maximum value.                                                               |
 | Missingness        | `expect_column_values_to_be_null`                         | Expect the column values to be null.                                                                                                   |
 | Missingness        | `expect_column_values_to_not_be_null`                     | Expect the column values to not be null.                                                                                               |
 | Numerical Data     | `expect_column_max_to_be_between`                         | Expect the column maximum to be between a minimum and a maximum value.                                                                 |
-| Numerical Data     | `expect_column_mean_to_be_between`                        | Expect the column mean to be between a minimum and a maximum value (inclusive).                                                        |
+| Numerical Data     | `expect_column_mean_to_be_between`                        | Expect the column mean to be between a minimum and a maximum value.                                                        |
 | Numerical Data     | `expect_column_median_to_be_between`                      | Expect the column median to be between a minimum and a maximum value.                                                                  |
 | Numerical Data     | `expect_column_min_to_be_between`                         | Expect the column minimum to be between a minimum value and a maximum value.                                                           |
-| Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length between a minimum value and a maximum value (inclusive).                           |
-| Pattern matching   | `expect_column_value_length_to_be_between`                | Expect the column entries to be strings with length between a minimum value and a maximum value (inclusive).                           |
+| Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length between a minimum value and a maximum value.                           |
```

Comment:
**note for reviewers** this expectation description was wrong, it looks like copy/pasta to me.